src/Goerli-DssSpell.sol

```diff
@@ -47,28 +71,145 @@ contract DssSpellAction is DssAction, DssSpellCollateralOnboardingAction {
         return false;
     }
 
-    address immutable MCD_FLAP = DssExecLib.flap();
-    address immutable MCD_ESM = DssExecLib.esm();
-
-    // No Transfer on Goerli
-    // address immutable MCD_GOV = DssExecLib.mkr();
-    // address immutable DUX_WALLET =        ;
-    // address immutable SIDESTREAM_WALLET = ;
-
     function actions() public override {
         // ---------------------------------------------------------------------
         // Includes changes from the DssSpellCollateralOnboardingAction
         // onboardNewCollaterals();
 
-        // ---------------------------- Lid for Flap ---------------------------
-        DssExecLib.setValue(MCD_FLAP, "lid", 30_000 * RAD);
 
-        // ------------------------------ ESM Min ------------------------------
-        DssExecLib.setValue(MCD_ESM, "min", 150_000 * WAD);
+        // Core Unit Budget DAI Transfer
+        // https://mips.makerdao.com/mips/details/MIP40c3SP67#budget-request-up-front
+        //
+        //    SH-001 - 230,000 DAI - 0xc657aC882Fb2D6CcF521801da39e910F8519508d
+        //  MAINNET ONLY
+
+
+        // Core Unit DAI Budget Stream
+        // https://mips.makerdao.com/mips/details/MIP40c3SP67#budget-request-up-front
+        //
+        //    SH-001 | 2022-06-01 to 2023-03-15 | 540,000 DAI | 0xc657aC882Fb2D6CcF521801da39e910F8519508d
+        // MAINNET ONLY
+
+
+        // Core Unit MKR Budget Stream
+        // https://mips.makerdao.com/mips/details/MIP40c3SP67#budget-request-up-front
+        //
+        //    SH-001 | 2022-06-01 to 2026-06-01 | Cliff 2023-11-23 | 250 MKR | 0x955993Df48b0458A01cfB5fd7DF5F5DCa6443550
+        // MAINNET ONLY
+
+
+        // MOMC Proposal
+        // https://vote.makerdao.com/polling/QmYx9e3k#poll-detail
+        //
+        // Maximum Debt Ceiling Decreases
+        //
+        //    Decrease WSTETH-A Maximum Debt Ceiling from 300 million to 200 million
+        DssExecLib.setIlkAutoLineDebtCeiling("WSTETH-A", 200 * MILLION);
+
+        //    Reduce Aave D3M Maximum Debt Ceiling from 300 million to 100 million
+        // MAINNET ONLY
+
+        //    Reduce LINK-A Maximum Debt Ceiling from 100 million DAI to 50 million DAI
+        DssExecLib.setIlkAutoLineDebtCeiling("LINK-A", 50 * MILLION);
+
+        // Maximum Debt Ceiling Increase
+        //
+        //    Increase MANA-A Maximum Debt Ceiling from 10 million DAI to 15 million DAI
+        DssExecLib.setIlkAutoLineDebtCeiling("MANA-A", 15 * MILLION);
+
+        // D3M Target Borrow Rate Decrease
+        //
+        //    Reduce DIRECT-AAVEV2-DAI Target Borrow Rate from 3.5% to 2.75%
+        // MAINNET ONLY
```

Comment:
I would probably add a comment about the D3M debt ceiling reduction and that it is only on mainnet

Reply:
Tracked at https://github.com/makerdao/spells-goerli/pull/78/files#diff-5ba6d6e31f0110bc3d9054811b3fd5bb9720eca165ac02046e44e3375c2f1d35R110